src/hooks/use-site-details.tsx

```diff
@@ -165,7 +169,7 @@ export function SiteDetailsProvider( { children }: SiteDetailsProviderProps ) {
 	);
 
 	const createSite = useCallback(
-		async ( path: string, siteName?: string ) => {
+		async ( path: string, siteName?: string, isImportingNewSite?: boolean ) => {
```

Comment:
Would be interesting initialize the parameter to false?

```suggestion
		async ( path: string, siteName?: string, isImportingNewSite: boolean = false ) => {
```

Reply:
Yeah makes sense, updated in https://github.com/Automattic/studio/pull/378/commits/11a8dd8a9d170230444a94b2c9a52d22449a8185